superset-frontend/.eslintrc.js

```diff
@@ -17,7 +17,12 @@
  * under the License.
  */
 module.exports = {
-  extends: ['airbnb', 'prettier', 'prettier/react'],
+  extends: [
+    'airbnb',
+    'prettier',
+    'prettier/react',
+    'plugin:react-hooks/recommended',
```

Comment:
`eslint-airbnb` has also `airbnb/hooks` to extend from, but it throws errors for inexhaustive deps.